ckanext/qdes_schema/plugin.py

```diff
@@ -22,6 +23,8 @@ class QDESSchemaPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IValidators)
     plugins.implements(plugins.IActions)
     plugins.implements(plugins.IPackageController, inherit=True)
+    plugins.implements(plugins.IFacets)
```

Comment:
Hi @awset - if we implement the `IFacets` interface with the `inherit=True` argument we don't need to implement the `group_facets` or `organization_facets` functions below - these can be removed:

```
    def group_facets(self, facets_dict, group_type, package_type):
        return facets_dict

    def organization_facets(self, facets_dict, organization_type, package_type):
        return facets_dict
```

Reply:
added inherit=true, thanks 👍